src/coreclr/src/System.Private.CoreLib/src/System/Reflection/CustomAttribute.cs

```diff
@@ -1374,7 +1374,7 @@ internal static bool IsAttributeDefined(RuntimeModule decoratedModule, int decor
                 }
                 else
                 {
-                    ctorWithParameters = ModuleHandle.ResolveMethodHandleInternal(decoratedModule.GetNativeHandle(), caCtorToken);
+                    ctorWithParameters = new ModuleHandle(decoratedModule.GetNativeHandle()).ResolveMethodHandle(caCtorToken).GetMethodInfo();
```

Comment:
```suggestion
                    ctorWithParameters = new ModuleHandle(decoratedModule).ResolveMethodHandle(caCtorToken).GetMethodInfo();
```
`RuntimeModule.GetNativeHandle` just returns `this`. It looks like a left over from some refactoring and can be deleted.

Reply:
Done